velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -955,5 +984,17 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     MinMaxByGroupByAggregationTest,
     testing::ValuesIn(getTestParams()));
 
+TEST_F(MinMaxByGroupByAggregationTest, largeValuesDate) {
+  auto vectors = {makeRowVector(
+      {makeConstant(Date(60577), 100), makeConstant(Date(-57604), 100)})};
```

Comment:
Please write something like 

auto parseDate = [](const std::string& dateStr) {
    Date returnDate;
    parseTo(dateStr, returnDate);
    return returnDate;
  };

and use this function to give the string representations of the dates you are using in the tests.

Reply:
Thanks for the suggestion, this test case is now removed and Date() tests are integrated with the rest of the aggregation test suite.